src/hacbs/components/Commits/commit-actions.ts

```diff
@@ -0,0 +1,12 @@
+import { Action } from '../../../shared/components/action-menu/types';
+import { Commit } from '../../types';
+
+export const useCommitActions = (commit: Commit): Action[] => {
+  return [
+    {
+      cta: () => {},
+      id: `view-${commit.metadata.name.toLowerCase()}`,
+      label: 'View',
```

Comment:
`Go to source` kebab action is outlined in the design doc. 
<img width="251" alt="image" src="https://user-images.githubusercontent.com/9964343/185913004-339f9f74-91c9-4438-bbaa-b8ee5c340a8e.png">